src/faq/plugin.ts

```diff
@@ -36,9 +36,9 @@ class FAQWidget extends Widget{
   protected onAfterAttach(msg: Message): void {
     if (!hasBeenAttatched) {
```

Comment:
Was there a reason you needed this `hasBeenAttached` flag? `onAfterAttach` should only fire when a node is attached (and it was previously *not* attached).